staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -216,21 +216,21 @@ func encodeObject(obj runtime.Object, gv schema.GroupVersion, serializer runtime
 }
 
 // LogAnnotation fills in the Annotations according to the key value pair.
-func LogAnnotation(ae *auditinternal.Event, key, value string, level auditinternal.Level) {
+func LogAnnotation(ae *auditinternal.Event, key string, value Annotation) {
```

Comment:
as my comment above, this Annotation concept is unnecessary and should not live in audit anyway.